### PR TITLE
provided the option to use customized span name

### PR DESCRIPTION
**Pull request by liangjiapei**
I would like to modify the span name to something else besides the boring resolver name.

In my own instance it, I have my OpentracingPlugin initialized with 
``createCustomSpanName: (name, info) => `resolver: ${info.parentType}.${name}` ``
so that the span name now becomes "resolver: TYPE.resolverName".

Any suggestion is welcomed. Thanks!

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -175,12 +180,14 @@ export default function OpentracingPlugin<InstanceContext extends SpanContext>({
               // idempotent method to add helpers to the first context available (which will be propagated by apollo)
               addContextHelpers(context);
 
-              const name = getFieldName(info);
+              let name = getFieldName(info);
```
Comment (DanielMSchmidt) [inline src/index.ts:183]: As the function is defaulted we can keep this as a const and remove line 189:

```suggestion
              const name = createCustomSpanName(getFieldName(info), info);
```

**Review by DanielMSchmidt — COMMENTED** [review]
Great idea, love it 👍 

Could you add a line about this new feature in the documentation?
I think it's small enough that we don't necessarily need a test for this, but if you happen to have the time I'd appreciate it!

**Review by DanielMSchmidt — APPROVED** [review]
Thanks a lot!

**Comment by liangjiapei** [comment]
@DanielMSchmidt When do you plan to include this feature within a new release?

**Comment by DanielMSchmidt** [comment]
@liangjiapei The release process is managed my semantic-release, seems like a token was invalid, I changed it and the CI run I just triggered should give us a new release with this PR included

**Comment by liangjiapei** [comment]
@DanielMSchmidt hmm, it seems that the release failed again with `SemanticReleaseError: No npm token specified.`again.

**Comment by liangjiapei** [comment]
@DanielMSchmidt Could you please check it out what the issue it is and maybe trigger another CI run to release the changes? Thanks!

**Comment by DanielMSchmidt** [comment]
On it 👍

**Comment by DanielMSchmidt** [comment]
Ah snap I found the problem: none of the commits were semantic commits, so they did not trigger a release. I'll add a check and add a commit to trigger a new build with this.

**Comment by DanielMSchmidt** [comment]
Although it's not listed the fix is included in [2.1.0](https://github.com/DanielMSchmidt/apollo-opentracing/releases/tag/v2.1.0).

Sorry for taking so long to release, from now on it's going to be a matter of minutes

**Pull request merged**